src/app/components/BibPage/BibDetails.jsx

```diff
@@ -72,6 +74,32 @@ const getDefinitionObject = (bibValues, fieldValue, fieldLinkable) => {
   );
 };
 
+const getOwner = (bib) => {
+  const items = bib.items;
+  const ownerArr = [];
+  let owner;
+
+  if (!items || !items.length) {
+    return null;
+  }
+
+  items.forEach(item => {
+    const ownerObj = item.owner && item.owner.length ?
+      item.owner[0].prefLabel : undefined;
+
+    ownerArr.push(ownerObj);
+  });
+
+  if (_every(ownerArr, (o) => (o === ownerArr[0]))) {
```

Comment:
Just want to make sure so if the bib and it's all items are belong to NYPL, we don't show the owner, and if some items are belong to us and some are not, the owner will be decided based on the first item's owner?